matcher/service/openreview_interface.py

```diff
@@ -73,11 +71,22 @@ def papers(self):
                             content_dict[key] = value
                         else:
                             self.logger.debug('Invalid filter provided in invitation: {}. Supported filter format "content.field_x=value1".'.format(element))
-            self.paper_notes = list(openreview.tools.iterget_notes(
+            all_papers = {note.id: note for note in openreview.tools.iterget_notes(
                 self.client,
                 invitation=paper_invitation,
-                content=content_dict))
-            self._papers = [n.id for n in self.paper_notes]
+                content=content_dict)}
+
+            if self.config_note.content.get('custom_user_demand_invitation'):
```

Comment:
`custom_user_demand_invitation` edges should override `demands` array not the paper list. The list of paper should always be built using the parameter `paper_invitation`

Reply:
Changed.